superset-frontend/src/chart/chartAction.js

```diff
@@ -361,38 +361,36 @@ export function exploreJSON(
 
     const chartDataRequestCaught = chartDataRequest
       .then(response => {
+        const queriesResponse = response.result;
+        
         if (isFeatureEnabled(FeatureFlag.GLOBAL_ASYNC_QUERIES)) {
           // deal with getChartDataRequest transforming the response data
           const result = 'result' in response ? response.result[0] : response;
           return dispatch(chartUpdateQueued(result, key));
```

Comment:
@robdiciuccio should we continue sending a single result to chartUpdateQuered action?

Reply:
Yes, the return value here is a [async job metadata payload](https://github.com/apache/incubator-superset/blob/4d329071a1651994574c3ad1f26e32b56d9ff810/superset/charts/schemas.py#L1108).